core/imgread/common.cpp

```diff
@@ -164,131 +171,98 @@ bool InitDrive_(wchar* fn)
 }
 
 #ifndef NOT_REICAST
-bool InitDrive(u32 fileflags)
+bool InitDrive(u32 fileflags/*=0*/)
 {
 	if (settings.imgread.LoadDefaultImage)
 	{
-		printf("Loading default image \"%s\"\n",settings.imgread.DefaultImage);
-		if (!InitDrive_(settings.imgread.DefaultImage))
+		printf("Loading default image \"%s\"\n", settings.imgread.DefaultImage);
+		if (!InitDrive_(settings.imgread.DefaultImage.c_str()))
 		{
-			msgboxf("Default image \"%s\" failed to load",MBX_ICONERROR);
+			msgboxf("Default image \"%s\" failed to load", MBX_ICONERROR);
 			return false;
 		}
 		else
```

Comment:
Remove the ``else`` please, it just creates clutter as it's unnecessary here.

Reply:
? looks fine to me? Why do you think it should be removed?

Reply:
Because it adds clutter and is unnecessary here.

````c
if (...)
{
  ...
  return false;
}
return true;
````
would be enough.
The ``if`` always returns, so the ``else`` is unnecessary